config/locales/cs.yml

```
@@ -225,184 +226,180 @@ cs:
+        notice: Tvé předplatné bylo zrušeno.
+        error: Při rušení tvého předplatného se něco nepovedlo. Zkus to znovu! Jestli problém přetrvává, prosím [pošli nám email na](mailto:info@crimethinc.com) abychom ti mohli pomoct.
 
       thanks:
-        heading: Thanks for your help!
+        heading: Díky za pomoc!
         description: |
-          We welcome your [feedback](mailto:info@crimethinc.com) about our efforts and your [participation](mailto:info@crimethinc.com) in them.
+          Uvítáme tvou [zpětnou vazbu](mailto:info@crimethinc.com) tykající se naší snahy a tvé [účasti](mailto:info@crimethinc.com) v ní.
+          Brzy ti přijde potvrzovací email.
 
-          You’ll receive an email confirmation soon.
-
-          If you signed up for a monthly subscription, you can update or cancel your subscription at any time by submitting your email address in the form at the bottom of the [support](/support) page.
+          Jestli si se přihlásil k měsíčnímu předplatnému, můžeš ho upravit nebo zrušit kdykoliv chceš vyplněním tvé emailové adresy ve formuláři na spodu stránky [Podpora](/support) page.
 
       mailers:
         edit_subscription:
-          subject: Request for Super Secret Support Link
-          header: Your Super Secret Support Link has arrived!
-          link_description: Anyone with the above link can use it to make changes to your subscription. It will expire in one hour. Keep it secret, keep it safe.
-          repeat_request: If you don't get around to making the changes you want in time, no worries. We can always send you another super secret link.
-          more_help: You can also reply to this email if need more assistance.
+          subject: Požádej o svů supertajný link na podporu
+          header:  Tvůj supertajný kod dorazil!
+          link_description: Kdokoliv kdo má přístup k odkazu nahoře, ho může použít k tomu aby změnil částku tvého předplatného. Za hodinu vyprší, udržuj ho v tajnosti a bezpečí
+          repeat_request: Jestli nestihneš co si potřeboval, není se čeho bát, vždy ti můžeme poslad další odkaz
+          more_help: Můžeš taky odpovědět na tento email jestli potřebuješ s něčím pomoct.
 
     languages:
-      view_site_in_locale:  View this site in Czech [here](https://cs.crimethinc.com).
-      view_tools_in_locale: You can access an array of posters, zines, and other materials in Czech [here](https://cs.crimethinc.com/tools).
-      view_books_in_locale: You can access our books in Czech [here](https://cs.crimethinc.com/books).
+      view_site_in_locale:  Podívej se na tuto stránku v angličtině [zde](https://crimethinc.com).
+      view_tools_in_locale: Tady najdeš řadu plakátů, zinů, a dalších materiálů v Angličtině  [zde](https://crimethinc.com/tools).
+      view_books_in_locale: Přístup k našim knihám v angličtině [zde](https://crimethinc.com/books).
```

Re: https://github.com/crimethinc/website/pull/1763/files#r486028533

Like @astronaut-wannabe suggested, these still need the subdomain.

```suggestion
      view_site_in_locale:  Podívej se na tuto stránku v angličtině [zde](https://cs.crimethinc.com).
      view_tools_in_locale: Tady najdeš řadu plakátů, zinů, a dalších materiálů v Angličtině  [zde](https://cs.crimethinc.com/tools).
      view_books_in_locale: Přístup k našim knihám v angličtině [zde](https://cs.crimethinc.com/books).
```